spec/ruby/core/objectspace/fixtures.rb

```diff
@@ -32,4 +32,29 @@ def self.scoped(wr)
     return Proc.new { wr.write "finalized"; wr.close }
   end
 
+  class ObjectToBeFound
+    attr_reader :id
+
+    def initialize(id)
+      @id = id
```

Comment:
Could this be renamed to `@name` instead?
`id` looks like `__id__`/`object_id`.